### ISSUE TITLE
Customizing Auto Startup After System Reboot

### ISSUE THREAD

**Issue by Hype365** [FezVrasta/ark-server-tools#899]
Currently, at least as far as I can tell from experience so far, when the system (OS, server, dedicated box, whatever you want to call it) restarts the Ark servers startup on their own. I am wondering if there is a way to adjust this behavior to 1) be able to toggle the auto-startup on and off at will and 2) tell it to only auto-start specific instances. 

I suppose there might be some information out there on this but my Google-fu appears to be lacking in power today and I wasn't able to find anything useful with my search terms lol

**Comment by klightspeed** [comment]
At the moment, assuming you use systemd, you would need to remove `arkmanager.service` from startup, and add the individual `arkmanager@<instance>.service` instances to startup.

**Comment by Hype365** [comment]
I apologize for being a Linux novice but I am unsure how to use systemd or even what that is

**Comment by klightspeed** [comment]
systemd is the default init manager in recent versions of Debian, CentOS
and Ubuntu.

To stop the arkmanager service, disable it, and have e.g. only the `main`
instance start on startup:

```
systemctl stop arkmanager.service
systemctl disable arkmanager.service
systemctl enable arkmanager@main.service
```

**Comment by Hype365** [comment]
so based on this I could stop and disable the service entirely if I wanted to as well and not have the servers auto start on reboot of system? I ask this because I am still in the setup stages for my server cluster and I don't have a need for the auto start currently while it's just me working on the system.

**Comment by klightspeed** [comment]
Yes, disabling the service will disable auto-start on server start.  It would then be up to you to either start the servers, or create an `@reboot` cronjob to do that.

**Comment by Hype365** [comment]
An @reboot cronjob.... hmm I'm sure I could google it but how would I set that up? I know how to do normal cronjob by m h dom mon and dow but haven't heard of @reboot.

**Comment by klightspeed** [comment]
One crontab guide is at <https://wiki.archlinux.org/index.php/cron>
